releasenotes/notes/orchestrator-core-check-86b88f9d933f31bb.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
```

Comment:
Does this change requires any change from users ? Will anything break on upgrade ?

If so we should explain what and how to mitigate it. If not, I think this should be a `feature` or `enhancements`.

Reply:
No it doesn't, changing this as `enhancements` 👍